src/vs/workbench/contrib/preferences/browser/settingsWidgets.ts

```diff
@@ -1372,7 +1372,7 @@ export class ObjectSettingCheckboxWidget extends AbstractListSettingWidget<IObje
 		rowElement.appendChild(element);
 
 		const valueElement = DOM.append(rowElement, $('.setting-list-object-value'));
-		valueElement.textContent = changedItem.key.data;
+		valueElement.textContent = item.keyDescription ? `${item.keyDescription} (${item.key.data})` : item.key.data;
```

Comment:
@rzhao271 fyi this allows me to show a nice label next to the checkbox:

![image](https://user-images.githubusercontent.com/900690/133577686-948f79a4-490f-4b20-8862-ae8803657925.png)